src/components/ModalPortal.js

```diff
@@ -119,9 +119,8 @@ export default class ModalPortal extends Component {
 
   beforeOpen() {
     const { appElement, ariaHideApp, bodyOpenClassName } = this.props;
-    refCount.add(bodyOpenClassName);
     // Add body class
-    elementClass(document.body).add(bodyOpenClassName);
+    bodyClassList.add(bodyOpenClassName);
```

Comment:
As this is iterated upon, it seems like we are moving in a direction very similar to the `elementClass` library that was here before. (The major under-the-hood difference being that `bodyOpenClassName` can be a string of multiple class names, and using the other helper `refCount` to keep track of opened modals relying on the class internally.)

If you would prefer, I could make `bodyClassList` look and feel similar to `elementClass`, i.e. have `document.body` be passed in as an argument like it is here. This would remove the line change here, thus keeping the "public API" of the helper class the same as how `elementClass` looked.